tap_adroll/client.py

```diff
@@ -23,16 +23,18 @@ def __init__(self, config_path, config):
             # Set expires_in to a negative number to force the client to reauthenticate
             'expires_in': '-30'
         }
-        extra = {
-            'client_id': config['client_id'],
-            'client_secret': config['client_secret']
-        }
+        #extra = {
+            #'client_id': config['client_id'],
+            #'client_secret': config['client_secret']
+        #}
```

Comment:
Can we remove this?

Reply:
This was accidentally committed, shouldn't be here.